blots/scroll.ts

```diff
@@ -185,9 +185,20 @@ class Scroll extends ScrollBlot {
         blockAttribute || this.statics.defaultChild.blotName,
         blockAttribute ? attributes[blockAttribute] : undefined,
       );
+      this.insertBefore(block, after);
```

Comment:
We have to insert before calling `block.insertContents(0, lineDelta)` as `Block#insertContents` needs `block.parent` to be present.